src/main/java/uk/gov/ons/ctp/response/action/service/ProcessEventService.java

```diff
@@ -220,7 +264,7 @@ private boolean processLetterCases(
    * @param status
    * @param instant
    */
-  private void createCaseActionEvent(
+  public void createCaseActionEvent(
```

Comment:
Any reason why you've changed these methods from private to public? As far as I can see they're only used by this class?

Reply:
I started doing refactoring and changed the access identifier. reverted